code/style.css

```diff
@@ -35,24 +94,94 @@ p {
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  position: relative;
 }
 
 .room-deck {
   width: 250px;
   height: 400px;
   background: midnightblue;
-
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  position: relative;
```

Comment:
On the killer-deck, weapon-deck and the room-deck you have the exact same code except the colors. Do you think you could solve that in a way with much less code? To incorporate them and just add the color separat?

Reply:
Maybe a common class?  
@nathalienordstrom Commenting on a multiple line selection is helpful here 👍